html/pfappserver/root/static.alt/src/views/Configuration/_components/BillingTiersList.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <b-card no-body>
+    <pf-config-list
+      :config="config"
+      :isLoading="isLoading"
+    >
+      <template slot="pageHeader">
+        <b-card-header><h4 class="mb-0" v-t="'Billing Tiers'"></h4></b-card-header>
+      </template>
+      <template slot="buttonAdd">
+        <b-button variant="outline-primary" :to="{ name: 'newBillingTier' }">{{ $t('Add Billing Tier') }}</b-button>
+      </template>
+      <template slot="emptySearch">
+        <pf-empty-table :isLoading="isLoading">{{ $t('No billing tiers found') }}</pf-empty-table>
+      </template>
+      <template slot="buttons" slot-scope="item">
+        <b-button size="sm" variant="outline-primary" :to="{ name: 'TODO' }" class="float-right">{{ $t('Traffic Shaping') }}</b-button>
+      </template>
```

Comment:
I'm not sure this button should be there. If we really need some button at the end of each row, we need to specify `pfConfigurationListColumns.buttons` in the list columns of the billing tiers.